pkg/ui/react-app/src/components/Checkbox.tsx

```diff
@@ -3,14 +3,18 @@ import { FormGroup, Label, Input, InputProps } from 'reactstrap';
 
 interface CheckboxProps extends InputProps {
   wrapperStyles?: CSSProperties;
+  isExplainCheckbox?: boolean;
 }
 
-const Checkbox: FC<CheckboxProps> = ({ children, wrapperStyles, id, ...rest }) => {
+const Checkbox: FC<CheckboxProps> = ({ children, wrapperStyles, id, disabled, isExplainCheckbox, ...rest }) => {
   return (
     <FormGroup className="custom-control custom-checkbox" style={wrapperStyles}>
-      <Input {...rest} id={id} type="checkbox" className="custom-control-input" />
+      <Input {...rest} id={id} type="checkbox" className="custom-control-input" disabled={disabled} />
       <Label style={{ userSelect: 'none' }} className="custom-control-label" for={id}>
         {children}
+        {isExplainCheckbox && disabled && (
+          <div className="popup-message">This functionality is only available when using the Thanos engine</div>
```

Comment:
Let's explain the feature too.
```suggestion
          <div className="popup-message">Explain allows you to view an expandable query plan of the PromQL query similar to SQL EXPLAIN. This functionality is only available when using the Thanos engine</div>
```